### ISSUE TITLE
At a certain range; Bulletproof Sheet is penetrable by any caliber

### ISSUE THREAD

**Issue by unknown-4o4** [studio-minus/ppg-bugs-and-requests#551]
**Describe the bug**
When at a certain close-range point, you can penetrate the bulletproof sheet with virtually any caliber.

**Clearly describe how to reproduce the bug**
Watch Video or: Place the sheet around 3.5 "tiles" away from the barrel. Note that it doesn't work while in slow motion.

**Mod list**
Tried with vanilla as well

**System info (please complete the following information)**
 - Operating system: Windows 10
 - Game version: 1.22.2
 - GPU: 1660ti
 - CPU: i5-9600kf

**Additional notes**
I am obviously aware that if you're almost point-blank; It's penetrable, however if you get to this exact distance and go slightly-
closer to the sheet while shooting, it stops penetrating.

Let me know if I'm missing something crucial or if this is not a bug.
--
https://user-images.githubusercontent.com/81907908/143410214-bb6f3068-f8af-47eb-814c-7705fbcc9f21.mp4

**Comment by mestiez** [comment]
This is very odd

**Comment by unknown-4o4** [comment]
Were you able to reproduce it in your game? I would like to know please.

**Comment by mestiez** [comment]
Yes. I can reproduce it.

**Comment by DindinYT37** [comment]
I thought this was a feature, since a bullet is stronger freshly shot than when shot from 150 meters?

**Comment by mestiez** [comment]
That is a feature, but this issue is describing the following: 

![image](https://user-images.githubusercontent.com/32131519/143662744-9f9bb20e-b7ca-43c6-9d69-16d97f533d8c.png)

This is weird and not intentional.

**Comment by unknown-4o4** [comment]
Also, if you didn't see: this bug does not work when in slow motion. Maybe this information can help you in any way?

**Comment by DindinYT37** [comment]
> Also, if you didn't see: this bug does not work when in slow motion. Maybe this information can help you in any way?

It's probably because of the game trying to predict the trajectory of the bullet because of not having enough time(/iterations). In Slow motion, the game of course has more time to calculate and doesn't have to predict.

Atleast that's my guess.

**Comment by mestiez** [comment]
> In Slow motion, the game of course has more time to calculate and doesn't have to predict.

Yes that's what it boils down to. The bullet trajectory is more precise in slow motion because there are more calculations per in-game second.

**Comment by unknown-4o4** [comment]
Ah okay, I thought it was mainly the physics iterations that was set and not the actual game speed, though I'm sure the physics iterations has a part?

**Comment by DindinYT37** [comment]
> Ah okay, I thought it was mainly the physics iterations that was set and not the actual game speed, though I'm sure the physics iterations has a part?

You see, "more calculations per in-game second" also include the physics iterations.

**Comment by unknown-4o4** [comment]
> > Ah okay, I thought it was mainly the physics iterations that was set and not the actual game speed, though I'm sure the physics iterations has a part?
> 
> You see, "more calculations per in-game second" also include the physics iterations.

Oh okay, the more I learn. Thanks
